clients/backup/backup.go

```diff
@@ -32,13 +32,11 @@ type Client struct {
 type Params struct {
 	// Organization containing TSM data to back up.
 	// If not set, all orgs will be included.
-	OrgID string
-	Org   string
+	clients.OrgParams
 
 	// Bucket containing TSM data to back up.
 	// If not set, all buckets within the org filter will be included.
-	BucketID string
-	Bucket   string
+	clients.BucketParams
```

Comment:
It could, honestly the only reason I chose to do it separately is because of the code comments in the struct fields, and because at least at the moment, `OrgBucketParams` doesn't possess any special member functions or extra fields so we didn't lose anything by leaving it as the two separate embeds